### PR TITLE
jackson 2.9.8 -> 2.9.9

### PR DESCRIPTION
**Pull request by sullis**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [ ] Local run of `mvn install` succeeds
- [ ] My code follows the code style of this project
- [ ] My change requires a change to the Javadoc documentation
- [ ] I have updated the Javadoc documentation accordingly
- [ ] I have read the **README** document
- [ ] I have added tests to cover my changes
- [ ] All new and existing tests passed
- [x] A short description of the change has been added to the **CHANGELOG**
- [ ] My change is to implement 1.11 parity feature and I have updated [LaunchChangelog](https://github.com/aws/aws-sdk-java-v2/blob/master/docs/LaunchChangelog.md)

## License

- [x] I confirm that this pull request can be released under the Apache 2 license

### CODE REVIEW

**Comment by varunnvs92** [comment]
There was a [report](https://github.com/spring-projects/spring-boot/issues/16910#issuecomment-494452323) of v2.9.9 introducing incompatible combination of versions. This might affect customers who use those dependencies. Holding off merging for further investigation.

**Comment by candrews** [comment]
IMHO, this version bump should be prioritized as it addresses a security issue, CVE-2019-12086: https://github.com/FasterXML/jackson-databind/issues/2326

File: .changes/next-release/bugfix-core-2a95227.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "category": "core",
```
Comment (dagnir) [inline .changes/next-release/bugfix-core-2a95227.json]: For core changes, please use 'AWS SDK for Java v2'

**Comment by Ristop** [comment]
Perhaps bump it to 2.9.9.1 to address https://nvd.nist.gov/vuln/detail/CVE-2019-12814

**Comment by sullis** [comment]
obsolete

**Pull request closed without merging**